lib/collection/collection.js

```diff
@@ -195,7 +195,7 @@ _.assign(Collection.prototype, /** @lends Collection.prototype */ {
 
         // move ids and stuff from root level to `info` object
         json.info = {
-            id: this.id,
```

Comment:
Any reason why this was removed?

Reply:
It's been replaced with `_posmtan_id`, the `id` field is not valid as per the `info` structure.